compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -2677,15 +2673,13 @@ fn extend_span_to_previous_binding(sess: &Session, binding_span: Span) -> Option
 /// use foo::{a, b::{c, d}};
 /// //       ^^^^^^^^^^^^^^^ -- true
 /// ```
+#[instrument(skip(sess))]
 fn find_span_immediately_after_crate_name(
     sess: &Session,
     module_name: Symbol,
     use_span: Span,
 ) -> (bool, Span) {
-    debug!(
-        "find_span_immediately_after_crate_name: module_name={:?} use_span={:?}",
-        module_name, use_span
-    );
+    debug!("find_span_immediately_after_crate_name",);
```

Comment:
there's no need to repeat the name of the function here and elsewhere; please remove.

Reply:
What do you think should be the message instead? Something like `debug!("entered")`?
(in the places where fields are added I can just remove the message)

Reply:
not totally certain that they're even adding much value to keep around at all.

Reply:
Removed those, then. (and the instrumentation for the containing fns)

Reply:
Why remove the `instrument` annotation. `#[instrument(level = "debug", skip(...))]` should "just work" without any dummy `debug!` statement in the body shouldn't it?

Reply:
Yes, you're absolutely right. I thought it only created spans but no "enter" event. I was also creating `info`-level spans by accident. Fixed.